addons/im_livechat/models/im_livechat_channel_member_history.py

```diff
@@ -187,6 +187,12 @@ def _compute_session_duration_hour(self):
     @api.model
     def action_open_discuss_channel_list_view(self, domain=()):
         discuss_channels = self.search_fetch(domain, ["channel_id"]).channel_id
+        if len(discuss_channels) == 1:
+            action = self.env["ir.actions.act_window"]._for_xml_id("im_livechat.discuss_channel_action")
```

Comment:
This line can be put below the `search_fetch` and removed from both places (L191, L196).